backends/advanced/src/advanced_omi_backend/workers/memory_jobs.py

```
@@ -127,176 +135,121 @@ async def process_memory_job(conversation_id: str, *, redis_client=None) -> Dict
+
+                # Determine memory provider from memory service
+                memory_provider = conversation_model.MemoryProvider.CHRONICLE  # Default
+                try:
+                    # Check for explicit provider identifier, fallback to class name
+                    provider_id = getattr(memory_service, "provider_identifier", None)
+                    if provider_id == "openmemory_mcp":
+                        memory_provider = conversation_model.MemoryProvider.OPENMEMORY_MCP
+                    elif not provider_id and "OpenMemory" in memory_service.__class__.__name__:
+                        memory_provider = conversation_model.MemoryProvider.OPENMEMORY_MCP
+                except Exception:
+                    pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)